gcsfs/core.py

```diff
@@ -798,7 +804,8 @@ async def _info(self, path, **kwargs):
             return await self._get_object(path)
         except FileNotFoundError:
             pass
-        out = await self._ls(path, detail=True, **kwargs)
+        kwargs['detail'] = True # Force to true for info
+        out = await self._ls(path, **kwargs)
```

Comment:
I was getting `got multiple values for keyword argument 'detail'` after making the `_get_object` change. Perhaps that's a sign of some other unexpected control flow change?